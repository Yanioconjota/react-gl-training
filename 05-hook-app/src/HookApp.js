import React from 'react';
import { Col } from "react-bootstrap";
//import PropTypes from "prop-types";
//import { useState } from "react";

const HookApp = () => {
  return (
    <Col xs={12}>
      <h1>Hola mundo!</h1>
    </Col>
  );
};

export default HookApp;