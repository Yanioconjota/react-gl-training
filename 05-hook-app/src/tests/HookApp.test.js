import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import HookApp from '../HookApp';

describe('Pruebas con HookApp', () => {
  test('el componente debe mostrarse correctamente', () => {
    const wrapper = shallow(<HookApp/>);
    expect(wrapper).toMatchSnapshot();
  });
  
});
