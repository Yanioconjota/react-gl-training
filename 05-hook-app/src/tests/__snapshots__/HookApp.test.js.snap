// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con HookApp el componente debe mostrarse correctamente 1`] = `
<Col
  xs={12}
>
  <h1>
    Hola mundo!
  </h1>
</Col>
`;
